src/header/common/last_event_id.rs

```diff
@@ -0,0 +1,63 @@
+use std::fmt::{self, Display};
+use header::{self, Header, Raw};
+
+/// `Last-Event-ID` header, defined in
+/// [RFC3864](https://html.spec.whatwg.org/multipage/references.html#refsRFC3864)
+///
+/// The `Last-Event-ID` header contains information about
+/// the last event in an http interaction so that it's easier to
+/// track of event state. This is helpful when working
+/// with [Server-Sent-Events](http://www.html5rocks.com/en/tutorials/eventsource/basics/). If the connection were to be dropped, for example, it'd
+/// be useful to let the server know what the last event you
+/// received was.
+///
+/// The spec is a String with the id of the last event, it can be
+/// an empty string which acts a sort of "reset".
+///
+/// # Example
+/// ```
+/// use hyper::header::{Headers, LastEventId};
+///
+/// let mut headers = Headers::new();
+/// headers.set(LastEventId("1".to_owned()));
+/// ```
+#[derive(Clone, Debug, PartialEq)]
+pub struct LastEventId(pub String);
```

Comment:
No idea. Looked on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers), [Wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields), and [IANA](https://www.iana.org/assignments/message-headers/message-headers.xhtml) but couldn't find a reference to this header. What list did you use to figure out what to include?